ntbs-service/Pages/Notifications/Edit/Patient.cshtml

```diff
@@ -213,11 +213,13 @@
                     </label>
                     <span nhs-span-type="ErrorMessage" id="birth-country-error"
                           ref="errorField" asp-validation-for="Patient.CountryId" has-error="@hasCountryError"></span>
-                    <select ref="selectField" v-on:change="validate" nhs-select-type="@countrySelectErrorState"
-                            asp-for="Patient.CountryId" asp-items="Model.Countries"
-                            data-aria-controls="year-of-entry-conditional">
-                        <option value="">Select a Birth Country</option>
-                    </select>
+                    <div ref="selectField" data-aria-controls="year-of-entry-conditional" class=@countrySelectErrorClass>
+                        <autocomplete-select v-on:validate-input="validate" placeholder="Select a Birth Country" validate="true" inline-template>
+                            <select asp-for="Patient.CountryId" asp-items="Model.Countries">
+                                <option value=""></option>
```

Comment:
We lose the `Select a Birth Country` option for non-javascript users in this instance. 
Docs suggest this route:
https://github.com/alphagov/accessible-autocomplete#null-options
(We should obviously dynamically add the default value if there is a database value for this already)

Reply:
I think I tried this and couldn't get it to work, the text "Select a Birth Country" was still present when that option was chosen. will check again though

Reply:
Just tried it again using the "defaultValue =''" setting as they say, but the placeholder text doesn't disappear when you click on it, which make it quite annoying to use if you're filling in a form for a new patient as you'll have to delete that whole line first :(

Reply:
Yeah that's not ideal... For anyone else reviewing here's an example of this:
https://alphagov.github.io/accessible-autocomplete/examples/
Go down to `default value Germany`.

If we're going with placeholder text I think we should add an SR only hint (class="govuk-visually-hidden" or something like that)?